language/sygus.tex

```diff
@@ -1290,116 +1284,103 @@ \subsection{Declaring Oracle Functional Symbols}
 \subsection{Pre-defined Oracle Types}
 Given a synthesis function symbol $F$ in the current signature
 with sort $\sigma_1, \ldots \sigma_n \rightarrow \sigma$, each of the following commands 
-is used to declare an external oracle with name $N$, which generate specific types of oracle constraints and assumptions over the behaviour of $F$. 
+is used to declare an external oracle with name $N$, which generate specific types of oracle constraints and assumptions over the behavior of $F$. 
 
 \begin{itemize}
 \item $\paren{\iooracledeckwd\mbox{ }F\mbox{ } N}$
 
-This declares an input-output oracle for function $F$. 
+This command declares an input-output oracle for function $F$. 
 It is syntactic sugar for:
 
-
-$\paren{\constraintoraclekwd\mbox{ } \paren{\paren{x_1\mbox{ }\sigma_1} \ldots \paren{x_n\mbox{ }\sigma_n}}\mbox{ }
-\paren{\paren{x \mbox{ }\sigma}}
-\paren{{\tt=}\mbox{ } \paren{F \mbox{ } x_1 \ldots x_n}\mbox{ }  x}\mbox{ }N
-}
-$
-
+\begin{lstlisting}[language=SyGuS-Desugar]
+($\constraintoraclekwd$ (($x_1$ $\sigma_1$) $\ldots$ ($x_n$ $\sigma_n$)) (($x$ $\sigma$)) (= ($F$ $x_1$ $\ldots$ $x_n$) $x$) $N$)
```

Comment:
should this have the same new line pattern as all the other oracle sugar? i.e., 

(inputs on one line)
(then outputs)
(then the constraint term)
then the binary name

Reply:
Thanks, good catch. Fixed it now